org.eclipse.jdt.ls.core/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -4,6 +4,7 @@ org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
 org.eclipse.jdt.core.compiler.compliance=1.8
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.release=disabled
```

Comment:
is this needed?

Reply:
It's not, it was automatically added by m2e on project import